test/smoke/dapi.js

```diff
@@ -1,10 +1,45 @@
+const DAPIClient = require('@dashevo/dapi-client');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(`DAPI ${nodeName}`, () => {
+      const options = {
+        debug: false,
+        verbose: false,
+        errors: false,
+        warnings: false,
+        seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
+      };
+      const dapi = new DAPIClient(options, 3000);
+      // getBlockHash height:1
+      it('should respond data from chain', async () => {
+        const blockHash = await dapi.getBlockHash(1);
+        // const response = await utils.requestWrapper(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/block-index/1`);
+        const resp = await fetch(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/block-index/1`)
```

Comment:
Why not directly from DashCore? As I know we are going to get rid of Insight soon.

Reply:
now use rpc.getBlockHash(1)